### DISCUSSION TITLE
DEM Gen

### DISCUSSION THREAD

**Discussion by bonaventuraadi** [gmtsar/gmtsar#776] [Q&A]
Hi, i would like to ask, why the dem.grd value from DEM Generator is different from SRTM 1 data that i downloaded from USGS or EarthData? Thank you for your answer.
![coba](https://github.com/gmtsar/gmtsar/assets/73235331/61cbae69-88a6-4c87-8c89-f1d7b68a74be)

**Comment by dsandwell** [comment]
The geoid. Insar meeds ellipsoidal heights.Sent from my iPhoneOn Aug 1, 2023, at 11:51 PM, bonaventuraadi ***@***.***> wrote:﻿
Reopened #776.

**Reply by bonaventuraadi** [reply]
Glad to know, thank you for your answer, Sir.
